### ISSUE TITLE
show relation members

### ISSUE THREAD

**Issue by andrewharvey** [osmlab/osm-deep-history#23]
I think it would be useful if the relation history view could also show members in the diff, to see how the members of a relation has changed over its history.

**Comment by sun-geo** [comment]
I agree, I would be also happy to have a second alternative to the great tool:
osm.mapki.com which does already show a relation member list history view.

Example: http://osm.mapki.com/history/relation.php?id=4846904
![image](https://user-images.githubusercontent.com/47290585/88303160-fe68cb80-cd06-11ea-8b80-80bb54398ac4.png)
